### DISCUSSION TITLE
Prevent fetching images instead of just blocking the loading?

### DISCUSSION THREAD

**Discussion by ghost** [seleniumbase/SeleniumBase#2710] [Q&A] [answered]
I'm running a SeleniumBase program through a proxy. On my initial test I used about 4.7mb of bandwith (on a page with many images). After setting the `block_images` parameter on`SB` I could visually confirm images were not loading but I am still using the same amount of bandwith.

Is `block_images` strictly for not displaying images on the DOM?

**Comment by mdmintz** [comment, marked as answer]
This is essentially what `block_images` does:

```python
from selenium import webdriver

chrome_options = webdriver.ChromeOptions()
prefs = {"profile.managed_default_content_settings.images": 2}
chrome_options.add_experimental_option("prefs", prefs)
driver = webdriver.Chrome(chrome_options=chrome_options)
```

Source: https://stackoverflow.com/a/31581387/7058266

It blocks the images from being displayed in the DOM.
It doesn't change bandwidth.
